lib/Onnxifi/onnxifiGlow.cpp

```diff
@@ -142,15 +142,30 @@ onnxGetBackendCompatibility(onnxBackendID backendID, size_t onnxModelSize,
     return ONNXIFI_STATUS_INVALID_SIZE;
   }
 
+  auto *glowBackendId =
+      reinterpret_cast<glow::onnxifi::BackendIdPtr>(backendID);
+  if (!glowBackendId) {
+    return ONNXIFI_STATUS_INVALID_POINTER;
+  }
+
   glow::Module M;
   auto *F = M.createFunction("check_compatibility");
-  std::unique_ptr<glow::ONNXModelLoader> loader =
-      glow::ONNXModelLoader::parse(onnxModel, onnxModelSize, *F);
+  std::unique_ptr<glow::onnxifi::ModelLoader> loader =
+      glow::onnxifi::ModelLoader::parse(onnxModel, onnxModelSize, *F);
 
+  // TODO: Make better error reporting.
   if (!loader) {
     return ONNXIFI_STATUS_UNSUPPORTED_OPERATOR;
   }
 
+  // Make sure that every node from the parsed function F
+  // can be executed by the specific backend.
+  for (const auto &op : F->getNodes()) {
```

Comment:
So model parse can be successful with unsupported ops inside the model?

Reply:
Yes, model parsing is generic and not specific to the backend.
Like we could add an Op support on Interpreter backend but do not support that on CPU backend for example.